src/routes/home/Home.js

```diff
@@ -1236,6 +1236,7 @@ class Home extends React.Component {
                 </label>
 
                 <Modal
+                  parentSelector={() => document.querySelector(`.${s.root}`)}
```

Comment:
Gah, this works in practice to put the toast above the modal, when running locally with `yarn start`, but `yarn test` is failing, see https://github.com/josephfrazier/reported-web/actions/runs/4379992906/jobs/7666550546:

```
FAIL src/routes/home/Home.test.js
  Home
    ✕ renders children correctly (50ms)

  ● Home › renders children correctly

    TypeError: Cannot read properties of null (reading 'appendChild')

      36 |
      37 |     const wrapper = renderer
    > 38 |       .create(
         |        ^
      39 |         <App context={{ insertCss: () => {}, fetch: () => {}, pathname: '' }}>
      40 |           <Home
      41 |             typeofcomplaintValues={typeofcomplaintValues}

      at Modal.componentDidMount (node_modules/react-modal/lib/components/Modal.js:104:14)
```

I think I need to figure out how to mock the `./Home.css` file when running Jest tests...

Reply:
worked around in 5ec43bb296ce5a9e4938335e6faaa1934065d7a6